app/front/scripts/services/visualizations/index.js

```diff
@@ -223,6 +223,57 @@ function paramsToBabbageStateRadar(params) {
   return result;
 }
 
+function formatValue(scale) {
+  if (_.isString(scale)) {
+    try {
+      scale = JSON.parse('{' + scale + '}');
+    } catch (e) {
+      scale = {};
```

Comment:
If we fail to parse, let's print something to the console.
Also, let's have the defaults set here and not above.